app/detail mock.html

```
@@ -0,0 +1,139 @@
+    </style>
+        <style>
+                body {
+                        min-height: max(884px, 100dvh);
+                }
+        </style>
+        <style>
+                body {
+                        min-height: max(884px, 100dvh);
+                }
+        </style>
+</head>
```

HTMLHint (1.9.2)

[error] <title> must be present in <head> tag.

(title-require)